### PR TITLE
Smartfridges only come preloaded at map start

### LINKED ISSUE #26981: Newly constructed chemistry smartfridges are already stocked.
For my own purposes of storing chemicals, i used a screwdriver to change the smartfridge board's type to chemical storage (like the one in chemistry) and when it was built it already had what supplies chemistry usually starts with inside it rather than being empty awaiting chemicals to be put in inside it.

> Uncertain if this is a abusable bug, helpful feature or general inconsistency.

### PR DESCRIPTION
**Pull request by coiax**
Fixes #26981.

:cl: coiax
fix: Smartfridges no longer magically gain medicines if deconstructed
and rebuilt.
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm [hunk 1]
```diff
@@ -384,34 +393,15 @@
...
 	return FALSE
 
-/obj/machinery/smartfridge/extract/New()
-	..()
-	var/obj/item/device/slime_scanner/I = new /obj/item/device/slime_scanner(src)
-	load(I)
-	var/obj/item/device/slime_scanner/T = new /obj/item/device/slime_scanner(src)
-	load(T)
+/obj/machinery/smartfridge/extract/preloaded
+	initial_contents = list(/obj/item/device/slime_scanner = 2)
```
Comment (QualityVan) [inline code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm]: Should you be calling ..() here?

Comment (coiax) [inline code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm]: It's not a proc, it's a type definition

Comment (QualityVan) [inline code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm]: Right.

**Comment by SgtxRootbeer** [comment]
then give us the ability to make the devices from rnd that come from smart fridges, for example, the slime scanner.

**Comment by coiax** [comment]
That's out of scope. Shouldn't be a hard PR though.

**Comment by XDTM** [comment]
couldn't it be based on the mapload argument instead of a subtype?

**Comment by coiax** [comment]
Eh, that's the other way to do it. Not sure if I like that way though.

**Pull request merged**